packages/kubernetes-1.26/kubelet-exec-start-conf

```diff
@@ -0,0 +1,31 @@
+[Service]
+ExecStart=
+ExecStart=/usr/bin/kubelet \
+{{#unless settings.kubernetes.standalone-mode}}
+    --cloud-provider "external" \
```

Comment:
@stmcginnis  Should I change it to `--cloud-provider {{default "external" settings.kubernetes.cloud-provider}} \` since the earlier versions of k8s should still be using in-tree providers would pick that up.

Reply:
I'm actually not sure what the value is supposed to be now with Kubernetes 1.26. We might need to ask someone from the EKS-D team.

Reply:
The right change would be migrate to `--cloud-provider external` and use empty string `""` if it's `metal-*`.